resource-service/README.md

```diff
@@ -18,46 +18,81 @@ limitations under the License.
 
 # Resource Service
 
-The Resource service accepts an incoming message from the `user` Kafka topic which contains the resourceId that is being accessed 
-(this could be an actual file, or a directory that could contain many files and/or sub-directories). The service will then query the backing store to
-see if the requested resourceId has been stored. If this is not the case then the request will be passed onto the local implementation of 
-the Resource service. All the returned resources will be within an Akka stream, each element in the stream is then consumed and
-added to the `resource` Kafka topic to be processed by the Policy service.
+> :warning:
+Windows users will have problems with Hadoop integration testing. 
+Included [here](src/component-tests/resources/hadoop-3.2.1/bin) is a Windows-compatible set of hadoop binaries. 
+To 'install' on Windows, copy the [hadoop.dll](src/component-tests/resources/hadoop-3.2.1/bin/hadoop.dll) to `C:\Windows\System32`. 
+This should enable the `HadoopResourceServiceTest` to run correctly. 
 
-## Message Model and Database Domain
+At it's core, the Resource Service is an implementation of the `ResourceService` interface:
+```java
+interface ResourceService {
+    Iterator<LeafResource> getResourcesById(final String resourceId);
+
+    Iterator<LeafResource> getResourcesByType(final String type);
+
+    Iterator<LeafResource> getResourcesBySerialisedFormat(final String serialisedFormat);
+
+    Boolean addResource(final LeafResource resource);
+}
+```
+
+This implementation is then wrapped with various service layers to work with the Palisade streaming architecture. 
+These additional layers include:
+* Pulling requests from an upstream kafka topic and writing responses to a downstream topic - see [AkkaRunnableGraph](src/main/java/uk/gov/gchq/palisade/service/resource/stream/config/AkkaRunnableGraph.java)
+* 'Caching' resources returned by previous `ResourceService` calls - see [ResourceServicePersistenceProxy](src/main/java/uk/gov/gchq/palisade/service/resource/service/ResourceServicePersistenceProxy.java)
+* Catching all errors and writing to a kafka error topic
+
+
+## Flow of Control
+
```

Comment:
I'm thinking that AkkaRunnableGraph is the heart of the service workflow and be the starting point of the discussion under a header Flow of Control.  Could be as simple as pulling the sentence about the Kafka topic into this section or starting this section with a few words on the subject

Reply:
Done